bazel/Dockerfile

```diff
@@ -1,18 +1,31 @@
 FROM ghcr.io/dependabot/dependabot-updater-core
 
-# TODO: Install any required dependencies for your ecosystem
-# Example:
-# RUN apt-get update \
-#   && apt-get install -y --no-install-recommends \
-#   your-package-manager \
-#   && rm -rf /var/lib/apt/lists/*
+USER root
+
+# Install Java and ca-certificates-java for Bazel (which uses embedded Java)
+RUN apt-get update && \
+    apt-get install -y --reinstall ca-certificates default-jre-headless ca-certificates-java && \
+    update-ca-certificates && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    curl -fsSL -o /usr/local/bin/bazelisk \
+    https://github.com/bazelbuild/bazelisk/releases/latest/download/bazelisk-linux-amd64 && \
+    chmod +x /usr/local/bin/bazelisk && \
+    ln -s /usr/local/bin/bazelisk /usr/local/bin/bazel
+
+# Create a system-wide .bazelrc to configure Bazel's JVM to use system certificates
+# Bazel ignores JAVA_TOOL_OPTIONS, so we must use startup options
+RUN echo "startup --host_jvm_args=-Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts" > /etc/bazel.bazelrc && \
+    echo "startup --host_jvm_args=-Djavax.net.ssl.trustStoreType=jks" >> /etc/bazel.bazelrc
```

Comment:
Is this factually correct?, `jks` is no longer the default type in newer Java versions

Is this property needed? I am asking because this may work only because Java ignores it.